### PR TITLE
[Components with more than 1 service] LPS-185248 Refactor Asset components from remaining asset modules

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**
Hello reviewer, PR removes the interface _GCloudNaturalLanguageDocumentAssetAutoTagger_ , OpenNLPDocumentAssetAutoTagger  because it was deprecated, in this Pull request => [https://github.com/brianchandotcom/liferay-portal/pull/74856](https://github.com/brianchandotcom/liferay-portal/pull/74856)

This ticket => [https://issues.liferay.com/browse/LPS-97123](https://issues.liferay.com/browse/LPS-97123)

### CODE REVIEW

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

File: modules/apps/asset/asset-auto-tagger-google-cloud-natural-language-impl/src/test/java/com/liferay/asset/auto/tagger/google/cloud/natural/language/internal/GCloudNaturalLanguageDocumentAssetAutoTaggerImplTest.java [hunk 1]
```diff
@@ -48,11 +48,11 @@ public class GCloudNaturalLanguageDocumentAssetAutoTaggerImplTest {
 
 	@BeforeClass
 	public static void setUpClass() {
-		_gCloudNaturalLanguageDocumentAssetAutoTaggerImpl =
-			new GCloudNaturalLanguageDocumentAssetAutoTaggerImpl();
+		_gCloudNaturalLanguageDocumentAssetAutoTagger =
```
Comment (rafaprax) [inline modules/apps/asset/asset-auto-tagger-google-cloud-natural-language-impl/src/test/java/com/liferay/asset/auto/tagger/google/cloud/natural/language/internal/GCloudNaturalLanguageDocumentAssetAutoTaggerImplTest.java:51]: @GabrielHenriquedev, please rename class to `GCloudNaturalLanguageDocumentAssetAutoTaggerTest`. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]
Hey @GabrielHenriquedev, please merge the 2nd and 3rd commits in one. Thx

**Comment by GabrielHenriquedev** [comment]
@rafaprax resent => [https://github.com/liferay-core-infra/liferay-portal/pull/4673](https://github.com/liferay-core-infra/liferay-portal/pull/4673)

**Pull request closed without merging**